### PR TITLE
56176 local laws and regulations

### PR DESCRIPTION
**Pull request by v-williamsw**
Made relatively minor updates to topic Paul had started.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [globalization/internationalization/international-laws-and-standards.md](https://github.com/MicrosoftDocs/globalization/blob/fbca85e0c274e3b896acff55128a5085de744269/globalization%2Finternationalization%2Finternational-laws-and-standards.md) | 96 | 100 | 92 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bea2f066-f0c6-4ba9-b908-9f3886f00b1e) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by Keika3** [comment]
I cannot seem to find a section or sentence for language reform that was requested in the task 56176 by Agustin. Will it be covered by another topic?

**Comment by v-williamsw** [comment]
Well-spotted Keika. I thought I had addressed Augustin's request, but clearly didn't. How about this, right after the "Language laws" section?:

Language reforms

Some markets have local regulatory boards that specify official spelling, grammar, and terminology for a given language. You should consider complying with these regulations. Information on markets affected are provided in the Language Reforms<https://microsoft.sharepoint.com/sites/globalreadiness/Lists/LanguageReform/Language%20Reform%20Information.aspx> section on the Global Readiness and Opportunity website.

It's pretty much cribbed (though condensed) from the SharePoint page Augustin links to in his comment in Task 56176, though it doesn't link to that "Assessing compliance with language reforms" page itself. Should I change it so it doesn't match any of the SharePoint content? (It does duplicate the first sentence there. This begs the question, should we never duplicate text found elsewhere in the Microsoft site?)

Thanks,

- Willie

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [globalization/internationalization/international-laws-and-standards.md](https://github.com/MicrosoftDocs/globalization/blob/c770bb22b8990a35f5c9e49a9bc0585c1162a2dd/globalization%2Finternationalization%2Finternational-laws-and-standards.md) | 96 | 100 | 92 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/778bf941-cbd2-46c5-9f66-05c69a96500d) | [:white_check_mark:](# "Successfully processed") |
| [globalization/toc.yml](https://github.com/MicrosoftDocs/globalization/blob/c770bb22b8990a35f5c9e49a9bc0585c1162a2dd/globalization%2Ftoc.yml) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c557414b-effd-4aba-8799-f9d8f84ced7d) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [globalization/internationalization/international-laws-and-standards.md](https://github.com/MicrosoftDocs/globalization/blob/d6e629eaf6cd145f3b456e2de977a1f9933bc0a4/globalization%2Finternationalization%2Finternational-laws-and-standards.md) | 94 | 100 | 81 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f892357f-b835-400c-a9f7-5cb9b2317c28) | [:white_check_mark:](# "Successfully processed") |
| [globalization/toc.yml](https://github.com/MicrosoftDocs/globalization/blob/d6e629eaf6cd145f3b456e2de977a1f9933bc0a4/globalization%2Ftoc.yml) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b6cf1edc-0f2e-42e1-833e-fdf88a8a36ab) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [globalization/internationalization/international-laws-and-standards.md](https://github.com/MicrosoftDocs/globalization/blob/9209696d777b99bb3d954b242aa4cdce688fda8c/globalization%2Finternationalization%2Finternational-laws-and-standards.md) | 92 | 100 | 77 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/830b9787-ee18-4982-b049-320774b081b1) | [:white_check_mark:](# "Successfully processed") |
| [globalization/toc.yml](https://github.com/MicrosoftDocs/globalization/blob/9209696d777b99bb3d954b242aa4cdce688fda8c/globalization%2Ftoc.yml) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/af34890d-0131-4652-8b4b-085a1b7af9f5) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [globalization/internationalization/international-laws-and-standards.md](https://github.com/MicrosoftDocs/globalization/blob/905cd2409262c8b5ed97448b722b19352fb864a1/globalization%2Finternationalization%2Finternational-laws-and-standards.md) | 91 | 100 | 76 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ea450aaa-2aec-4892-946a-8695e87ef29b) | [:white_check_mark:](# "Successfully processed") |
| [globalization/toc.yml](https://github.com/MicrosoftDocs/globalization/blob/905cd2409262c8b5ed97448b722b19352fb864a1/globalization%2Ftoc.yml) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/28e5dab0-8ce3-44db-a02b-7525a090a933) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [globalization/internationalization/international-laws-and-standards.md](https://github.com/MicrosoftDocs/globalization/blob/dcf8b163f583df9bc8e75af2a28d8dc2610e737e/globalization%2Finternationalization%2Finternational-laws-and-standards.md) | 91 | 100 | 76 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cc3c4be6-fcc6-4053-9d6a-938161b1c768) | [:white_check_mark:](# "Successfully processed") |
| [globalization/toc.yml](https://github.com/MicrosoftDocs/globalization/blob/dcf8b163f583df9bc8e75af2a28d8dc2610e737e/globalization%2Ftoc.yml) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8225b429-df47-4dfe-9ae9-36e46e8174d1) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: globalization/internationalization/international-laws-and-standards.md [hunk 1]
```diff
@@ -1,8 +1,71 @@
...
+Cryptography laws vary by country.
+Some countries ban encryption in communication. Other countries require a license to use encrypted software, or require that police have decryption keys in case they're needed in an investigation.
+
+For example, although encryption in France is perfectly legal, it's subject to strict government control regarding its distribution (supply), import/export, and usage. Your product may need to submit the necessary approvals to the French government prior to shipping to France.
+
+[World map of encryption laws and policies](https://www.gp-digital.org/world-map-of-encryption/ "Encryption laws and policies in different countries") gives the laws and policies of different countries.
+
+## Language laws
+
+[*La Loi de Toubon*](https://en.wikipedia.org/wiki/Toubon_Law) (Toubon Law) applies to products and services sold in France, no matter which country they're from.
```
Comment (jowilco) [inline globalization/internationalization/international-laws-and-standards.md:38]: Let's replace this link with https://www.legifrance.gouv.fr/loda/id/LEGITEXT000005616341/
and change the link text to "La Loi de Toubon (Toubon Law)"

File: globalization/internationalization/international-laws-and-standards.md [hunk 2]
```diff
@@ -1,8 +1,71 @@
...
+
+## Language reforms
+
+Some markets have local regulatory boards that specify official spelling, grammar, and terminology for a given language. You should consider complying with these regulations. For example, Reform der deutschen Rechtschreibung von 1996 (German orthography reform of 1996) led to a mandate of the use use of new spelling rules in schools and public administration. For more information about German orthography, see [Rat für deutsche Rechtschreibung (Council for German Orthography)](https://www.rechtschreibrat.com).
+
+## Entry to market
+
+Some markets may require you to register your software before it can be distributed or sold in that market.
+The perquisites and processes vary by market.
+For example, China requires that software and services support the national standard: [GB18030 code page](https://en.wikipedia.org/wiki/GB_18030).
```
Comment (jowilco) [inline globalization/internationalization/international-laws-and-standards.md:62]: https://icu-project.org/docs/papers/gb18030.html

File: globalization/internationalization/international-laws-and-standards.md [hunk 3]
```diff
@@ -1,8 +1,71 @@
...
+
+## Laws or standards particular to your program
+
+For applications that target specific functionality—such as taxes or health care—keep in mind that different laws and standards apply and your program may be liable to adhere to those laws.
+For example, you may need to conform to the US [Health Insurance Portability & Accountability Act (HIPAA)](https://en.wikipedia.org/wiki/Health_Insurance_Portability_and_Accountability_Act) regulations, the European Union [General Data Protection Regulation (GDPR)](https://eur-lex.europa.eu/legal-content/TXT/?uri=CELEX:32018R1725) regulations, or the Canadian [Personal Information Protection and Electronic Documents Act (PIPEDA)](https://www.priv.gc.ca/en/privacy-topics/privacy-laws-in-canada/the-personal-information-protection-and-electronic-documents-act-pipeda/) regulations depending on target markets.
```
Comment (jowilco) [inline globalization/internationalization/international-laws-and-standards.md:71]: https://www.govinfo.gov/app/details/PLAW-104publ191

File: globalization/internationalization/international-laws-and-standards.md [hunk 4]
```diff
@@ -1,8 +1,71 @@
 ---
 title: International laws and standards
-description: A global application sold to international markets must conform to the laws and standards of each market.
+description: If your software is distributed globally, then it must conform to all local laws. To understand your risks and obligations, you should consult an expert. Below are some basic guidelines to get you started.
 ---
```
Comment (jowilco) [inline globalization/internationalization/international-laws-and-standards.md:4]: add
ms.date: 8/12/2022
author: v-williamsw

File: globalization/internationalization/international-laws-and-standards.md [hunk 5]
```diff
@@ -1,8 +1,71 @@
...
 title: International laws and standards
-description: A global application sold to international markets must conform to the laws and standards of each market.
+description: If your software is distributed globally, then it must conform to all local laws. To understand your risks and obligations, you should consult an expert. Below are some basic guidelines to get you started.
```
Comment (jowilco) [inline globalization/internationalization/international-laws-and-standards.md:3]: Delete "Below are some basic guidelines to get you started."

File: globalization/internationalization/international-laws-and-standards.md [hunk 6]
```diff
@@ -1,8 +1,71 @@
...
+Here are some basic guidelines to get you started.
+
+## Security
+
+With the proliferation of support for languages, scripts, and glyphs, software internationalization has a direct impact on security. For example, you might consider validating user-supplied URLs to avoid IDN homograph attacks. To learn about the security engineering practices used at Microsoft to build and operate highly secure apps and services, see [Microsoft Security Engineering](https://www.microsoft.com/securityengineering).
+
+## Privacy and international law
+
+Privacy is an important consideration in many countries. For example, the General Data Protection Regulation (GDPR) in the European Union (EU) applies to anyone doing business in or with the EU. See [EUR-Lex -32018R1725](https://eur-lex.europa.eu/legal-content/TXT/?uri=CELEX:32018R1725) for official versions of the regulation. For information about GDPR, and for information about how to honor rights and fulfill GDPR obligations when using Microsoft products and services, see [General Data Protection Regulation Summary](/compliance/regulatory/gdpr).
```
Comment (jowilco) [inline globalization/internationalization/international-laws-and-standards.md:17]: delete "honor rights and "

File: globalization/internationalization/international-laws-and-standards.md [hunk 7]
```diff
@@ -1,8 +1,71 @@
...
+In many markets, you may be subject to legal action if you violate local laws.
+Some countries have signed international copyright laws, such as the [TRIPS Agreement 1994](https://www.wto.org/english/tratop_e/trips_e/trips_e.htm "Agreement on Trade-Related Aspects of Intellectual Property Rights") and/or the [Berne Convention 1928](https://www.wipo.int/treaties/en/ip/berne/ "Berne Convention for the Protection of Literary and Artistic Works").
+If using another entity's components—open-source or not—it's important to understand the license agreement associated with that code and to adhere to that agreement.
+See [Intellectual property and open innovation](https://www.microsoft.com/legal/intellectualproperty "Legal aspects of intellectual property and open innovation") to learn about the multiple modes of intellectual property and open innovation systems that Microsoft relies on.
+
+## Encryption declaration
+
+Cryptography laws vary by country.
+Some countries ban encryption in communication. Other countries require a license to use encrypted software, or require that police have decryption keys in case they're needed in an investigation.
+
```
Comment (jowilco) [inline globalization/internationalization/international-laws-and-standards.md:31]: Can delete this new line. I don't think we need new paras.

File: globalization/internationalization/international-laws-and-standards.md [hunk 8]
```diff
@@ -1,8 +1,71 @@
...
+For example, although encryption in France is perfectly legal, it's subject to strict government control regarding its distribution (supply), import/export, and usage. Your product may need to submit the necessary approvals to the French government prior to shipping to France.
+
+[World map of encryption laws and policies](https://www.gp-digital.org/world-map-of-encryption/ "Encryption laws and policies in different countries") gives the laws and policies of different countries.
```
Comment (jowilco) [inline globalization/internationalization/international-laws-and-standards.md:34]: I think that we should avoid this link. It is a nice site, but it is also a private 3rd party, Let's just remove this sentence.

File: globalization/internationalization/international-laws-and-standards.md [hunk 9]
```diff
@@ -1,8 +1,71 @@
...
+If using another entity's components—open-source or not—it's important to understand the license agreement associated with that code and to adhere to that agreement.
+See [Intellectual property and open innovation](https://www.microsoft.com/legal/intellectualproperty "Legal aspects of intellectual property and open innovation") to learn about the multiple modes of intellectual property and open innovation systems that Microsoft relies on.
+
+## Encryption declaration
+
+Cryptography laws vary by country.
+Some countries ban encryption in communication. Other countries require a license to use encrypted software, or require that police have decryption keys in case they're needed in an investigation.
+
+For example, although encryption in France is perfectly legal, it's subject to strict government control regarding its distribution (supply), import/export, and usage. Your product may need to submit the necessary approvals to the French government prior to shipping to France.
```
Comment (jowilco) [inline globalization/internationalization/international-laws-and-standards.md:32]: Add link to https://www.ssi.gouv.fr/en/regulation/cryptology/

File: globalization/internationalization/international-laws-and-standards.md [hunk 10]
```diff
@@ -1,8 +1,71 @@
...
+
+- marketing and presentation materials (labels, leaflets, catalogs, brochures, order forms, product sheets, delivery slips)
+- instructions for use (whether on paper, audio, through an online resource, or built into the software)
+- descriptions of the extent and the conditions of guarantee
+- announcements (including advertising) intended for the general public
+
+In Canada by its constitution, both French and English are official languages. In addition to the requirement that government services be available in both languages, there are further laws that require specific materials to be available in both languages. For example, the [Safe Food for Canadians Regulations](https://laws-lois.justice.gc.ca/eng/regulations/SOR-2018-108/) requires that labels on food must be shown in both languages. In the province of Québec, French was declared the official language by Bill 96. See [National Assembly of Québec: Bill 96](http://m.assnat.qc.ca/en/travaux-parlementaires/projets-loi/projet-loi-96-42-1.html "An Act respecting French, the official and common language of Québec") for an explanation of the scope of where French is to be used in Québec.
```
Comment (jowilco) [inline globalization/internationalization/international-laws-and-standards.md:46]: Replace "In Canada by its constitution, both French and English are official languages." with "English and French are a fundamental characteristic of the Canadian identity, and the importance of language rights is clearly recognized in the Canadian Charter of Rights and Freedoms, part of the Constitution Act of 1982."

File: globalization/internationalization/international-laws-and-standards.md [hunk 11]
```diff
@@ -1,8 +1,71 @@
...
+- descriptions of the extent and the conditions of guarantee
+- announcements (including advertising) intended for the general public
+
+In Canada by its constitution, both French and English are official languages. In addition to the requirement that government services be available in both languages, there are further laws that require specific materials to be available in both languages. For example, the [Safe Food for Canadians Regulations](https://laws-lois.justice.gc.ca/eng/regulations/SOR-2018-108/) requires that labels on food must be shown in both languages. In the province of Québec, French was declared the official language by Bill 96. See [National Assembly of Québec: Bill 96](http://m.assnat.qc.ca/en/travaux-parlementaires/projets-loi/projet-loi-96-42-1.html "An Act respecting French, the official and common language of Québec") for an explanation of the scope of where French is to be used in Québec.  
+
+English and French are a fundamental characteristic of the Canadian identity, and the importance of language rights is clearly recognized in the Canadian Charter of Rights and Freedoms, part of the Constitution Act of 1982.
```
Comment (jowilco) [inline globalization/internationalization/international-laws-and-standards.md:48]: Delete 48 and 48. 
Add 50 to 46

File: globalization/internationalization/international-laws-and-standards.md [hunk 12]
```diff
@@ -1,8 +1,71 @@
...
+There are laws requiring access to goods or services in both official languages.
+For more information, please see [About official languages and bilingualism](https://www.canada.ca/en/canadian-heritage/services/official-languages-bilingualism/about.html).
+
+Other markets have their own laws and new laws are added by countries periodically.
+
+## Language reforms
+
+Some markets have local regulatory boards that specify official spelling, grammar, and terminology for a given language. You should consider complying with these regulations. For example, Reform der deutschen Rechtschreibung von 1996 (German orthography reform of 1996) led to a mandate of the use use of new spelling rules in schools and public administration. For more information about German orthography, see [Rat für deutsche Rechtschreibung (Council for German Orthography)](https://www.rechtschreibrat.com).
```
Comment (jowilco) [inline globalization/internationalization/international-laws-and-standards.md:56]: Delete "You should consider complying with these regulations. "

File: globalization/internationalization/international-laws-and-standards.md (hunk 10 above)
Comment (jowilco) [reply globalization/internationalization/international-laws-and-standards.md:46]: Do we want to keep with Paul's practice of having each sentence on a new line? I really don't see any advantages to doing that.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [globalization/internationalization/international-laws-and-standards.md](https://github.com/MicrosoftDocs/globalization/blob/617716354c11c53bc88178a4bb5ca7a07bbdfd34/globalization%2Finternationalization%2Finternational-laws-and-standards.md) | 91 | 100 | 75 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bf8c5427-8efa-4760-bb82-a0cb5cca618a) | [:white_check_mark:](# "Successfully processed") |
| [globalization/toc.yml](https://github.com/MicrosoftDocs/globalization/blob/617716354c11c53bc88178a4bb5ca7a07bbdfd34/globalization%2Ftoc.yml) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1008e423-b918-4221-b4f7-65e627c07d77) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: globalization/internationalization/international-laws-and-standards.md [hunk 13]
```diff
@@ -1,8 +1,55 @@
...
+- announcements (including advertising) intended for the general public
+
+English and French are a fundamental characteristic of the Canadian identity, and the importance of language rights is clearly recognized in the Canadian Charter of Rights and Freedoms, part of the Constitution Act of 1982. In addition to the requirement that government services be available in both languages, there are further laws that require specific materials to be available in both languages. For example, the [Safe Food for Canadians Regulations](https://laws-lois.justice.gc.ca/eng/regulations/SOR-2018-108/) requires that labels on food must be shown in both languages. For more information, please see [About official languages and bilingualism](https://www.canada.ca/en/canadian-heritage/services/official-languages-bilingualism/about.html). In the province of Québec, French was declared the official language by Bill 96. See [National Assembly of Québec: Bill 96](http://m.assnat.qc.ca/en/travaux-parlementaires/projets-loi/projet-loi-96-42-1.html "An Act respecting French, the official and common language of Québec") for an explanation of the scope of where French is to be used in Québec.
```
Comment (jowilco) [inline globalization/internationalization/international-laws-and-standards.md:37]: delete "please" from "For more information, please see"

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [globalization/internationalization/international-laws-and-standards.md](https://github.com/MicrosoftDocs/globalization/blob/9a0f73730df3011a62fea7982adfd5c272f0af9c/globalization%2Finternationalization%2Finternational-laws-and-standards.md) | 93 | 100 | 80 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bf6116ba-1cb5-43fe-9aff-c274459d3622) | [:white_check_mark:](# "Successfully processed") |
| [globalization/toc.yml](https://github.com/MicrosoftDocs/globalization/blob/9a0f73730df3011a62fea7982adfd5c272f0af9c/globalization%2Ftoc.yml) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a017b112-538e-4bb5-9ffd-4952acdea443) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Review by jowilco — APPROVED** [review]

**Review by Keika3 — APPROVED** [review]

**Pull request merged**